2.10/2.10.22/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 2.10.22
+Сделать без цикла.
+Выводится всегда 4 конферма.
+Если хоть один конферм вернул фолс, то в конце выводится алерт: "Проиграл" иначе алерт: "Выиграл".
+<script>
+	let hasMistake = false;
```

Comment:
'hasError' is preferable because the word 'mistake' means an issue in a word, but not in an app